Dockerfile.release

```diff
@@ -0,0 +1,12 @@
+FROM alpine:3.5
+
+WORKDIR /go/src/app
```

Comment:
You don't need this.. @NitishT also you can follow Dockerfile.release 

```
     apk add --no-cache ca-certificates && \
     apk add --no-cache --virtual .build-deps curl && \
```

We need ca-certificates as well. 
```
     echo 'hosts: files mdns4_minimal [NOTFOUND=return] dns mdns4' >> /etc/nsswitch.conf && \
```